pennylane/device_tracker.py

```diff
@@ -0,0 +1,171 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+from collections import defaultdict
+
+import pennylane as qml
+
+
+def track(dev, version="default", **kwargs):
+    r"""Creates a tracking context and applies it to a device.
+
+    Args:
+        dev (~.Device): a PennyLane-compatible device
```

Comment:
```suggestion
        dev (Device): a PennyLane-compatible device
```
I think the keyword comes up without the `~.`. Currently it's not showing:
![image](https://user-images.githubusercontent.com/49409390/121692205-de183b80-ca95-11eb-9926-055502a334af.png)